extensions/kubectl-cli/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "kubectl-cli-tool",
+  "displayName": "kubectl CLI Tool for Podman Desktop",
+  "description": "Install and update kubectl CLI Tools without leaving Podman Desktop",
+  "version": "0.0.1",
+  "icon": "icon.png",
+  "publisher": "redhat",
+  "license": "Apache-2.0",
+  "engines": {
+    "podman-desktop": "^0.0.1"
+  },
+  "main": "./dist/extension.js",
+  "contributes": {
```

Comment:
This is not implemented and should be removed. Only cli.registerCliTool() call should be used to register tools